### DISCUSSION TITLE
컨트리뷰터로 등록하기

### DISCUSSION THREAD

**Discussion by juunini** [juunini/alcohol-per-hour#5] [Announcements]
# 콜라보레이터 등록 요청을 하는 곳입니다.

이슈에 글을 등록했거나, 이슈에서 커뮤니케이션을 한 뒤
aph에 관심이 생겨 컨트리뷰터가 되고싶으시면
이 곳에 댓글로 신청해주시기 바랍니다.

신청하시면 콜라보레이터로 초대해드립니다.
(이메일을 확인하셔야 합니다.)
컨트리뷰터로 등록되는 조건은 아래와 같습니다.

- 이슈에 유의미한 의견을 올렸다.
- 이슈에서 유의미한 커뮤니케이션을 했다.

매우 주관적인 조건이지만 판단하는 쪽도 주관적이라 괜찮습니다(?)  

콜라보레이터로 등록되고나면 [CONTRIBUTORS.md](https://github.com/juunini/alcohol-per-hour/blob/main/CONTRIBUTORS.md) 에 자기 계정을 등록해서 직접 컨트리뷰터가 되어주세요.

**Comment by Sihan-Son** [comment]
컨티리뷰터가 되고 싶습니다

**Reply by juunini** [reply]
초대 완료

**Comment by Huansock** [comment]
컨트리뷰터가 되고 싶습니다.
